index.html

```diff
@@ -0,0 +1,144 @@
+                <img src="./assets/polymer_icon.svg" />
+                <div class="tech-heading">Polymer</div>
+                <div class="tech-release">Release: 2015</div>
+                <div class="tech-description">Polymer is an open-source JavaScript library for building web applications using Web Components.</div>
+            </div>
+        </div>
+    </div>
+  </div>
+
+  <!-- BACKEND TOOLS SECTION -->
+  <div class="title">
+    <img src="./assets/triangle.svg" />
+    <span>Back-End Tools</span>
+  </div>
+
+  <div class="parent-card">
+
+      <div class="card">
+        <div class="outer-tech">
+            <div class="tech">
+                <img src="./assets/rails_icon.png" />
+                <div class="tech-heading">Ruby on Rails</div>
+                <div class="tech-release">Release: 2005</div>
+                <div class="tech-description">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby.Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</div>
+            </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="outer-tech">
+            <div class="tech">
+                <img src="./assets/express_icon.svg" />
+                <div class="tech-heading">NodeJS</div>
+                <div class="tech-release">Release: 2009</div>
+                <div class="tech-description">Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event driven, non-blocking I/O model to make it lightweight and efficient.</div>
+            </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="outer-tech">
+            <div class="tech">
+                <img src="./assets/aspnet_icon.png" />
+                <div class="tech-heading">ASP.NET</div>
+                <div class="tech-release">Release: 2016</div>
+                <div class="tech-description">ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.</div>
+            </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="outer-tech">
+            <div class="tech">
+                <img src="./assets/django_icon.svg" />
+                <div class="tech-heading">Django</div>
+                <div class="tech-release">Release: 2005</div>
+                <div class="tech-description">Django (/ˈdʒæŋɡoʊ/ JANG-goh) is a free and open-source web framework written in Python, which follows the model-view-template(MVT) architectural pattern.</div>
+            </div>
+        </div>
+      </div>
+
+  </div>
+
+  <!-- FOOTER SECTION -->
+  <div class="footer">
+    <div class="item-1">WE ARE GAME CHANGERS</div>
+    <img src="./assets/wolox_logo_white.svg"/>
+    <div class="item-2">Copyright 2018. Wolox. All rights reserved</div>
+  </div>
+
+</body>
+</html>
```

Comment:
an empty line should be at the bottom of every file. add it here

Reply:
pushed the changes